_data/sidebars/lb4_sidebar.yml

```diff
@@ -15,13 +15,13 @@ children:
   url: Installation.html
   output: 'web, pdf'
 
-- title: 'Getting started'
-  url: Getting-started.html
+- title: 'Tutorial'
+  url: Tutorial.html
   output: 'web, pdf'
   children:
 
-  - title: 'Examples and tutorials'
-    url: Examples-and-tutorials.html
+  - title: 'More examples and tutorials'
+    url: More-examples-and-tutorials.html
```

Comment:
This will break any pages referring to the old URL `/More-examples-and-tutorials.html`. Can we preserve the old URL please?

Reply:
My reason for renaming this URL is that preserving the current name is confusing UX.
The bar ends up looking like this:
```
+ Tutorial
+ Examples and Tutorials
```

If we're going to separate them to distinguish between our default introductory tutorial and our other resources, we should make sure there's no confusion about what is what.

What if we rename the route to something like "Additional Learning"?

Reply:
What about `other examples and tutorials`? 
Or rename the `Tutorial` to something else? e.g. step-by-step tutorial?

Reply:
IMO, the URL does not necessarily have to be the same as the page title, e.g. we can have "More examples and tutorials" at `lb4/Examples-and-tutorials.html`. Although I agree it's better when they match each other.

The reason why I am concerned about this change is that my upcoming blog post is referring to `Examples-and-tutorials.html` URL and I'd rather avoid synchronization between landing my PR adding the blog post and this PR breaking old URLs.

On the second thought, I think we should simply leverage [jekyll-redirect-from plugin](https://github.com/jekyll/jekyll-redirect-from#usage) that's already configured for loopback.io and add redirect from `Examples-and-tutorials.html` to `More-examples-and-tutorials.html`.

Reply:
How about making `Examples and Tutorials` a top level item listing all examples / tutorials including the Garden Path tutorial ... and under it the first item can be the Garden Path Tutorial. We can link directly to the `Tutorial` page from `Getting-Started`?